### PR TITLE
enable podman-remote pod rm

### PR DESCRIPTION
**Pull request by baude**
add the ability to delete a pod from the remote client.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: cmd/podman/pod.go [hunk 1]
```diff
@@ -18,7 +18,13 @@ var podCommand = cliconfig.PodmanCommand{
 	},
 }
 
+//imageSubCommands are implemented both in local and remote clients
```
Comment (haircommander) [inline cmd/podman/pod.go]: s/image/pod/g :)

File: libpod/runtime_pod.go [hunk 2]
```diff
@@ -181,3 +181,27 @@ func (r *Runtime) GetRunningPods() ([]*Pod, error) {
 	}
 	return runningPods, nil
 }
+
+// GetPodsByContext gets pods whether all, latest, or a slice of names/ids
+func (r *Runtime) GetPodsByContext(all, latest bool, pods []string) ([]*Pod, error) {
```
Comment (mheon) [inline libpod/runtime_pod.go]: Can we avoid adding this? I really don't want to overpopulate libpod with endpoints to add containers/pods. This one seems to belong in cmd/podman/shared or libpod/adapter - it's very much a podman specific thing

Comment (baude) [reply libpod/runtime_pod.go]: moved to adapter/shortcuts

**Comment by TomSweeneyRedHat** [comment]
No luck tonight @baude, another merge victim.

**Comment by mheon** [comment]
LGTM
@vrothberg @rhatdan @TomSweeneyRedHat PTAL

**Comment by vrothberg** [comment]
/lgtm

Also tested locally :+1:

**Pull request merged**